react/react1/week3/todo-list/src/components/Todolist.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useCallback } from 'react';
+import '../Style.css';
+
+export default function Todolist({ todolist, onDeleteToDo }) {
+  const [isEdit, setIsEdit] = useState([]);
```

Comment:
This one seems to be used as a boolean but is defined as an empty array to start with?